person.rb

```diff
@@ -1,14 +1,15 @@
 require './nameable'
 
 class Person
```

Comment:
- [ ] Remember above :arrow_up: we got mentioned that a rental belongs to a person(person.rentals << self). Now, we would require a method for adding rentals in this person's class. Also kindly note that the person itself should be sent as a parameter to create the new rental achieving the has many associations. [Helpful link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)

Reply:
Well resolved ✔️